addons/mail/static/src/new/composer/emoji_picker.xml

```diff
@@ -24,7 +24,7 @@
                         <t t-esc="current"/>
                     </div>
                 </t>
-                <span class="o-emoji p-1 cursor-pointer" t-att-title="emoji.name" t-att-data-index="emoji_index" t-att-data-codepoints="emoji.codepoints">
+                <span class="o-emoji p-1 cursor-pointer" t-att-title="emoji.name" t-att-data-index="emoji_index" t-att-data-codepoints="emoji.codepoints" t-att-data-category="category.sortId">
```

Comment:
`category` is not necessarily defined, notably if there is search.
This is immediately related to the crash I mentioned just recently.